docs/content/developer/getting-started/connect.mdx

```diff
@@ -154,6 +154,12 @@ iota client switch --env <EnvAlias>
 The latest stable release. As such, it has the latest features that were tested in the `development` branch of the
```

Comment:
Not directly related to this PR, but the branch is called `develop` and not `development`

Reply:
Thanks, fixed in 3c02fcdb2af574de07480bf868489f183623912c